src/style.css

```diff
@@ -208,6 +208,7 @@
 }
 
 .rct9k-marker {
+  z-index: 999;
```

Comment:
Suntem in cazul https://redmine.xops-online.com/issues/34055

Daca ma uit peste n luni la ac chestie parasutata, nu imi va inspira nimic. De comentat sixfacut trimitere la acel(e) locuri care au legatura. E.g. modificarea de mai jos.